pkg/handlers/v1/cloud_insert.go

```diff
@@ -16,6 +16,7 @@ type CloudAssetChanges struct {
 	AccountID    string            `json:"accountId"`
 	Region       string            `json:"region"`
 	ResourceID   string            `json:"resourceId"`
+	ARN          string            `json:"arn"`
```

Comment:
ResourceID should be unique enough. What was the reason for adding this bit of information?

If we do want to use this, we will require changes to the upstream services, the SH schema, and the docs. Could you create a ticket or tickets to address this?

Reply:
I could be convinced either way.  Ben and I both thought ARN would be better, and I think he's already working on the other services to pass it through. We should discuss..

Reply:
I don't have any strong opinions either way as longs as all the necessary things get updated

Reply:
Did we end up making tickets for this work?

Reply:
No, but I'll follow up.

Reply:
SECD-481 opened